include/libtorrent/aux_/hasher512.hpp

```diff
@@ -58,6 +58,14 @@ POSSIBILITY OF SUCH DAMAGE.
 	#include <openssl/sha.h>
 	}
 
+#elif defined TORRENT_USE_WOLFCRYPT
+
+// when using wolfcrypt, we need to enable the SHA-512 hasher
+#define WOLFSSL_SHA512 1
+
+#define WC_NO_HARDEN
```

Comment:
could you add a comment explaining what this does?

Reply:
will do